src/platform/backends/shared/linux/backend_utils.cpp

```diff
@@ -86,6 +87,34 @@ auto virtual_switch_subnet(const QString& bridge_name)
     }
     return subnet.toStdString();
 }
+
+void check_min_img_size(const mp::MemorySize& requested_size, const mp::Path& image_path)
+{
+    auto qemuimg_process = mp::ProcessFactory::instance().create_process(
+        std::make_unique<mp::QemuImgProcessSpec>(QStringList{"info", image_path}));
+    auto process_state = qemuimg_process->execute();
+
+    if (!process_state.completed_successfully())
+        throw std::runtime_error(fmt::format("Cannot get image info: qemu-img failed ({}) with output:\n{}",
+                                             process_state.failure_message(),
+                                             qemuimg_process->read_all_standard_error()));
+
+    const auto img_info = QString{qemuimg_process->read_all_standard_output()};
+    const auto capture_name = QStringLiteral("size");
```

Comment:
This seems a bit excessive… I don't think repeating "size" would be that big of a deal here, and having to substitute this in the pattern below… Your call.

Reply:
OK, I will do that.

Reply:
Bahh I totally forgot about this and just noticed this PR never got merged, ![](https://is.gd/LwcLnA)